syntax = "proto2";
package qqnews;

import "profile.proto";

message LuaNewsInfo {
    required uint32          lua_id    = 1;
    optional KBUserInfo      user_info = 2;
    optional uint32          test_field = 3;
}
